src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ChoiceGroup/BitChoiceGroupDemo.razor

```diff
@@ -85,4 +86,62 @@
         </ExamplePreview>
     </ComponentExampleBox>
 
-</ComponentDemo>
\ No newline at end of file
+
+    <ComponentExampleBox Title="ChoiceGroup with custom options" HTMLSourceCode="@example8HtmlCode" CSharpSourceCode="@example8CSharpCode" ExampleId="example8">
+        <ExamplePreview>
+            <div class="example-desc">ChoiceGroup options are customized by LabelTemplate property in the ChoiceGroupOption class</div>
+            <BitChoiceGroup Label="Pick one" Options="Example8Options">
+            </BitChoiceGroup>
+        </ExamplePreview>
+    </ComponentExampleBox>
+
+</ComponentDemo>
+
+
+@code {
+
+    // example 8
+
+    public static RenderFragment<BitChoiceGroupOption> LabelTemplateSample { get; set; }
```

Comment:
Why did you use the `static` property?

Reply:
<img width="783" alt="Untitled" src="https://user-images.githubusercontent.com/62190682/164522738-46a60596-6bff-4ef6-8934-1b55b2ee5c1c.png">

This is the reason.
However, we have another solution. To change this property to a field and define it in the constructor.
what do you think? @Cyrus-Sushiant

Reply:
it not importatnt, it is ok this way.

Reply:
there is no right or wrong here. it is not an actual real situation we have here. it's a special case that we have in the demo pages that will not (and should not) occur in a real project (using such a long property initialization in a class).